src/components/PageSizeSelector/index.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "@emotion/styled";
+import { Select, Option } from "@leafygreen-ui/select";
+import { PAGE_SIZES } from "constants/index";
+import usePageSizeSelector from "./usePageSizeSelector";
+
+interface Props {
+  value: number;
+  onChange?: (i: number) => void;
```

Comment:
nit. Since it seems like the `onChange` prop is always being called, I don't think it should be optional